### ISSUE TITLE
React.__spread is not a function after upgrade to React 15.0.0

### ISSUE THREAD

**Issue by wilsont** [skratchdot/react-bootstrap-daterangepicker#70]
Just found out this error when I upgrade my project to react15.0.0

```javascript
render: function () {
		this.setOptionsFromProps();
		return React.createElement(
			'div',
			React.__spread({ref: 'picker'},  this.props),
			this.props.children
		);
}
```

**Comment by carlsagan21** [comment]
I made a PR about it. 
Check and use it for a time... It worked for me at least. :)
#72

**Comment by skratchdot** [comment]
fixed in commit 0498756b0df1d741fd6c3e19b71202da045a0552  I published 3.0.0 which can be used with React 15.  Thanks for checking it out!
